### DISCUSSION TITLE
[Requirement] Setting - port number

### DISCUSSION THREAD

**Discussion by eoureo** [coddingtonbear/obsidian-web#86] [Ideas]
Thanks for the great program.
Obsidian Web is very useful Extension.

I have a requirement.

Please add a setting to change port numbers for multiple Obsidian Vaults.

http://127.0.0.1:27123/*
https://127.0.0.1:27124/*

I want to replace 27123 and 27124 with other numbers.

**Comment by coddingtonbear** [comment]
Unfortunately, allowing that to be configured introduces some complexities that I'm not comfortable taking on at the moment; please see #71.

**Reply by eoureo** [reply]
Thank you.
